### PR TITLE
Add Community Sample Directory

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes #(issue-number)

## Proposed Changes
The sample directory should be for clear and maintained samples.
- Moved non-silver language hello-world samples to new community sample directory.
- Added documentation to point to new directory.

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -49,6 +50,24 @@ project was started by Google but has contributions from a growing number of
 industry-leading companies. For a current list of the authors, see
 [Authors](https://github.com/knative/serving/blob/master/AUTHORS).
 
+### Authoring samples
+
+Beyond the official documentation there are endless possibilities for combining
+tools, platforms, languages and products. By submitting a tutorial you can
```
Comment (samodell) [inline community/README.md]:
```suggestion
tools, platforms, languages, and products. By submitting a tutorial you can
```

File: serving/samples/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ more about Knative Serving resources.
...
+| Hello World                | A quick introduction that highlights how to deploy an app using Knative Serving.                                                                                                                                         | [C#](helloworld-csharp/README.md), [Go](helloworld-go/README.md), [Java](helloworld-java/README.md), [Kotlin](helloworld-kotlin/README.md), [Node.js](helloworld-nodejs/README.md), [PHP](helloworld-php/README.md), [Python](helloworld-python/README.md), [Ruby](helloworld-ruby/README.md) |
```
Comment (samodell) [inline serving/samples/README.md]: Should the Kotlin sample also be moved into the community folder?

File: community/samples/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Knative Community Samples
```
Comment (samodell) [inline community/samples/README.md]: Can we add links to the community samples from here, similar to the /serving/samples/README.md?

File: serving/samples/README.md (hunk 2 above)
Comment (averikitsch) [reply serving/samples/README.md]: Kotlin is on the rise and Google is starting to support it, so I thought I'd keep it.

Comment (samodell) [reply serving/samples/README.md]: SGTM, then!

File: community/samples/README.md (hunk 3 above)
Comment (averikitsch) [reply community/samples/README.md]: done.

File: community/README.md (hunk 1 above)
Comment (averikitsch) [reply community/README.md]: done.

File: community/samples/README.md (hunk 3 above)
Comment (samodell) [reply community/samples/README.md]: Thanks!

**Pull request merged**